src/Nncase.Core/IR/Call.cs

```diff
@@ -45,6 +45,11 @@ public sealed record Call(Expr Target, IRArray<Expr> Parameters) : Expr, IParame
     /// </summary>
     public Tuple<List<DataType>, List<List<QuantParam>>, float> EnodeBestQuantConfigWithCosine;
 
+    /// <summary>
+    /// adaround tmp output.
+    /// </summary>
+    public Tensor AdaRoundOutput;
```

Comment:
Do not add tensor in call, When tensor value is modified, Call will not guarantee it's equivalence.